linux/Jenkinsfile

```diff
@@ -253,10 +253,11 @@ def jenkinsStepAlpine() {
         apkArchAllList = ['x86_64', 'aarch64']
     }
     // remove aarch64 for JDK20 and below
-    if (${VERSION} -lt 21) {
+    def versionInt = VERSION as int // Parse Version As Integer
```

Comment:
isnt it better to do
`def versionInt = VERSION.toInteger()`
instead?

Reply:
I think this is subjective, from a quick google "as Integer" seems the favorite, unless there's a specific reason?

Reply:
I tried this, and it caused gradle to break, so I've reverted it back to my original format.